No se puede crear un combo de descuentos para productos

Pasos para reproducir el error:
- Abrir el menú descuentos
- Clic en la pestaña Combos
- Clic en 'Añadir combo'
No sucede nada. Internamente, se ve este trace:

```
Hibernate: select usuario0_.id as id1_36_, usuario0_.activo as activo2_36_, usuario0_.apellidoMaterno as apellido3_36_, usuario0_.apellidoPaterno as apellido4_36_, usuario0_.celular as celular5_36_, usuario0_.correo as correo6_36_, usuario0_.dni as dni7_36_, usuario0_.intereses as interese8_36_, usuario0_.nombres as nombres9_36_, usuario0_.password as passwor10_36_, usuario0_.perfil_id as perfil_12_36_, usuario0_.telefono as telefon11_36_, usuario0_.tienda_id as tienda_13_36_ from Usuario usuario0_
Jun 18, 2018 6:23:47 PM javafx.fxml.FXMLLoader$ValueElement processValue
WARNING: Loading FXML document with JavaFX API of version 8.0.141 by JavaFX runtime of version 8.0.121
Hibernate: select productode0_.id as id1_25_, productode0_.activo as activo2_25_, productode0_.codCupon as codCupon3_25_, productode0_.duracionDias as duracion4_25_, productode0_.fechaFin as fechaFin5_25_, productode0_.fechaInicio as fechaIni6_25_, productode0_.producto_id as product10_25_, productode0_.stockMaximo as stockMax7_25_, productode0_.valorDescuento as valorDes8_25_, productode0_.valorPct as valorPct9_25_ from ProductoDescuento productode0_
Hibernate: select producto0_.id as id1_21_0_, producto0_.activo as activo2_21_0_, producto0_.categoria_id as categor12_21_0_, producto0_.descripcion as descripc3_21_0_, producto0_.fragilidad_id as fragili13_21_0_, producto0_.imagen as imagen4_21_0_, producto0_.nombre as nombre5_21_0_, producto0_.peso as peso6_21_0_, producto0_.precio as precio7_21_0_, producto0_.precioCompra as precioCo8_21_0_, producto0_.stockFisico as stockFis9_21_0_, producto0_.stockLogico as stockLo10_21_0_, producto0_.volumen as volumen11_21_0_, productoca1_.id as id1_23_1_, productoca1_.activo as activo2_23_1_, productoca1_.descripcion as descripc3_23_1_, productoca1_.nombre as nombre4_23_1_, productofr2_.id as id1_26_2_, productofr2_.descripcion as descripc2_26_2_, productofr2_.productofragilidad as producto3_26_2_ from Producto producto0_ left outer join ProductoCategoria productoca1_ on producto0_.categoria_id=productoca1_.id left outer join ProductoFragilidad productofr2_ on producto0_.fragilidad_id=productofr2_.id where producto0_.id=?
Hibernate: select productoca0_.id as id1_24_, productoca0_.activo as activo2_24_, productoca0_.categoria_id as categori6_24_, productoca0_.fechaFin as fechaFin3_24_, productoca0_.fechaInicio as fechaIni4_24_, productoca0_.value as value5_24_ from ProductoCategoriaDescuento productoca0_
Hibernate: select productoca0_.id as id1_23_0_, productoca0_.activo as activo2_23_0_, productoca0_.descripcion as descripc3_23_0_, productoca0_.nombre as nombre4_23_0_ from ProductoCategoria productoca0_ where productoca0_.id=?
Hibernate: select clientedes0_.id as id1_1_, clientedes0_.activo as activo2_1_, clientedes0_.condicion as condicio3_1_, clientedes0_.fechaFin as fechaFin4_1_, clientedes0_.fechaInicio as fechaIni5_1_, clientedes0_.tipo as tipo6_1_, clientedes0_.value as value7_1_ from ClienteDescuento clientedes0_
Hibernate: select combopromo0_.id as id1_5_, combopromo0_.activo as activo2_5_, combopromo0_.descripcion as descripc3_5_, combopromo0_.duracionDias as duracion4_5_, combopromo0_.fechaFin as fechaFin5_5_, combopromo0_.fechaInicio as fechaIni6_5_, combopromo0_.imagen as imagen7_5_, combopromo0_.maxDisponible as maxDispo8_5_, combopromo0_.nombre as nombre9_5_, combopromo0_.numVendidos as numVend10_5_, combopromo0_.preciounireal as preciou11_5_, combopromo0_.preciounitario as preciou12_5_, combopromo0_.volumen as volumen13_5_ from ComboPromocion combopromo0_
Exception in thread "JavaFX Application Thread" java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at javafx.fxml.FXMLLoader$MethodHandler.invoke(FXMLLoader.java:1774)
	at javafx.fxml.FXMLLoader$ControllerMethodEventHandler.handle(FXMLLoader.java:1657)
	at com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(CompositeEventHandler.java:86)
	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:238)
	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:191)
	at com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(CompositeEventDispatcher.java:59)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:58)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.EventUtil.fireEventImpl(EventUtil.java:74)
	at com.sun.javafx.event.EventUtil.fireEvent(EventUtil.java:54)
	at javafx.event.Event.fireEvent(Event.java:198)
	at javafx.scene.Scene$ClickGenerator.postProcess(Scene.java:3470)
	at javafx.scene.Scene$ClickGenerator.access$8100(Scene.java:3398)
	at javafx.scene.Scene$MouseHandler.process(Scene.java:3766)
	at javafx.scene.Scene$MouseHandler.access$1500(Scene.java:3485)
	at javafx.scene.Scene.impl_processMouseEvent(Scene.java:1762)
	at javafx.scene.Scene$ScenePeerListener.mouseEvent(Scene.java:2494)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:381)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler.lambda$handleMouseEvent$354(GlassViewEventHandler.java:417)
	at com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock(QuantumToolkit.java:389)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler.handleMouseEvent(GlassViewEventHandler.java:416)
	at com.sun.glass.ui.View.handleMouseEvent(View.java:555)
	at com.sun.glass.ui.View.notifyMouse(View.java:937)
	at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
	at com.sun.glass.ui.gtk.GtkApplication.lambda$null$49(GtkApplication.java:139)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at sun.reflect.misc.Trampoline.invoke(MethodUtil.java:71)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at sun.reflect.misc.MethodUtil.invoke(MethodUtil.java:275)
	at javafx.fxml.FXMLLoader$MethodHandler.invoke(FXMLLoader.java:1769)
	... 33 more
Caused by: java.lang.IllegalStateException: Location is not set.
	at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2434)
	at javafx.fxml.FXMLLoader.load(FXMLLoader.java:2409)
	at com.sigad.sigad.descuentos.controller.MantenimientoDescuentosController.editRegistrarComboPromocion(MantenimientoDescuentosController.java:438)
	at com.sigad.sigad.descuentos.controller.MantenimientoDescuentosController.gotAgregarCombo(MantenimientoDescuentosController.java:470)
	... 43 more
```

--- a/src/main/java/com/sigad/sigad/descuentos/controller/RegistrarComboProductosController.java
+++ b/src/main/java/com/sigad/sigad/descuentos/controller/RegistrarComboProductosController.java
@@ -60,7 +60,7 @@ public class RegistrarComboProductosController implements Initializable {
     /**
      * Initializes the controller class.
      */
-    public static final String viewPath = "/com/sigad/sigad/descuentos/view/registrarComboProductos.fxml";
+    public static final String viewPath = "/com/sigad/sigad/descuentos/view/RegistrarComboProductos.fxml";
 
     @FXML
     private Label lblError;

